examples/python/tools/exa-tool/chile_government_services_assistant_.py

```
@@ -0,0 +1,141 @@
+# !pip install flask firecrawl praisonaiagents google-genai python-dotenv deep-translator
+
+# """# Set API Keys"""
+
+import os
+
+os.environ['FIRECRAWL_API_KEY'] = "your api key here"
+os.environ['OPENAI_API_KEY'] = "your api key here"
+
+# """# Import Libraries & Translator"""
+
+from firecrawl import FirecrawlApp, ScrapeOptions
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)